Index.html

```diff
@@ -0,0 +1,44 @@
+
+<!--  -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
```

Comment:
We shouldn't add a style sheet link if we don't also have the style sheet in this commit